docs/examples/ConnectionlessOffer.md

```diff
@@ -1,28 +1,153 @@
+        },
+        "metaRetries": 5
+      }
+      ```
+
+3. The `Admin` forwards the `invitationUrl` on to the `Peer`
+
+4. The `Peer` checks the validity of the `invitationUrl` using `agent.parseInvitation`
+
+      ```typescript
+      const oob = await agent.parseInvitation(invitationUrl)
+      ```
+
+5. The `Peer` accepts the Invitation using `agent.acceptInvitation`
+
+      ```typescript
+      await agent.acceptInvitation(oob)
+      ```
+
+6. The `Issuer` will then send back the credential offer.
+7. The `Peer` listens for the credential offer response from the `Issuer`, and accepts it
+
+    ```typescript
+    agent.addListener(SDK.ListenerKey.MESSAGE, async (newMessages: SDK.Domain.Message[]) => {
+      // newMessages can contain any didcomm message that is received, including
+      // Credential Offers, Issued credentials and Request Presentation Messages
+      const credentialOffers = newMessages.filter(
+        (message) => message.piuri === 'https://didcomm.org/issue-credential/3.0/offer-credential'
+      )
+
+      if (credentialOffers.length) {
+        for (const credentialOfferMessage of credentialOffers) {
+          try {
+            // Create credential offer object from the message
+            const credentialOffer = await SDK.OfferCredential.fromMessage(credentialOfferMessage)
+
+            // Prepare the credential request
+            const requestCredential = await agent.prepareRequestCredentialWithIssuer(credentialOffer)
+
+            // Send the credential request
+            await agent.sendMessage(requestCredential.makeMessage())
+          } catch (err) {
+            console.error('Error occurred while sending the credential request:', err)
+          }
+        }
+      }
+    })
+    ```
+
+8. The `Issuer` sends back the `Verifiable Credential`
+
+9. The `Peer`listens for the credential and stores it in pluto
+
+  ```typescript
+  agent.addListener(SDK.ListenerKey.MESSAGE, async (newMessages: SDK.Domain.Message[]) => {
+    // newMessages can contain any didcomm message that is received, including
+    // Credential Offers, Issued credentials and Request Presentation Messages
+    const issuedCredentials = newMessages.filter(
+      (message) => message.piuri === 'https://didcomm.org/issue-credential/3.0/issue-credential'
```

Comment:
Propose to replace it from
'https://didcomm.org/issue-credential/3.0/issue-credential'
to
SDK.ProtocolType.DidcommIssueCredential